Makefile

```diff
@@ -164,7 +164,8 @@ install:
 .PHONY: develop
 # Install Streamlit as links in your Python environment, pointing to local workspace.
 develop:
-	cd lib ; python setup.py develop
+	cd lib; \
+		pipenv install --skip-lock --sequential -e .
```

Comment:
Wouldn't a `pipenv install` or `pipenv install --skip --sequential` also work since the editable install is already included in the actual Pipfile?

Reply:
ah, yeah, that should work -- changed